lib/msal-browser/docs/working-with-b2c.md

```diff
@@ -0,0 +1,104 @@
+# Working with MSAL.js and Azure AD B2C
+
+> :warning: Before you start here, make sure you understand [how to initialize an app object](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/initialization.md) and [working with resources and scopes](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/resources-and-scopes.md). We also recommend general familiarity with **Azure AD B2C**. See the [B2C documentation](https://docs.microsoft.com/azure/active-directory-b2c/technical-overview) for more.
```

Comment:
Should we not instead link to the docs article for the initialization?

https://docs.microsoft.com/en-us/azure/active-directory/develop/msal-js-initializing-client-applications

Reply:
I did link the initialization docs in the repository just above. I have the impression that its better if these docs refer to each other within the repo.